include/wx/cmndata.h

```diff
@@ -209,6 +216,8 @@ class WXDLLIMPEXP_CORE wxPrintDialogData: public wxObject
     bool            m_printEnablePageNumbers;
     bool            m_printEnableHelp;
     bool            m_printEnablePrintToFile;
+    bool            m_printEnableCurrentPage;
```

Comment:
Again, for consistency this one should be
```suggestion
    bool            m_printEnableSelectCurrentPage;
```